### DISCUSSION TITLE
关于"增量编译导致修改html文件后未触发#[html_sql]重新编译"的一个提议

### DISCUSSION THREAD

**Discussion by gy0801151351** [rbatis/rbatis#173] [Ideas]
![image](https://user-images.githubusercontent.com/16450354/153025150-836b9af9-0243-43ed-8df0-ff4460b6afa3.png)

如上图介绍所示, 在仅修改html文件后不能触发重新编译的问题. 经过rust 1.58 版本的测试, 可以用 include_str!() 或 include_bytes!() 将html文件的内容静态导入当前的rs文件就能解决这个问题. 
       但是, 总感觉这样有些怪怪的. 所以, 在这里提议, 能否让 #[html_sql] 宏支持结合 include_str!() 或 include_bytes!() 一起使用呢? 再说明白一点, 就是能否让 #[html_sql] 宏从 include_str!() 或 include_bytes!() 的内容中获取 html内容, 不是必须地要再次读取html文件呢 ?

**Comment by zhuxiujia** [comment]
v3.1版本开始，需要开启debug_mode 才会启用该特性
